pkg/dbrole/dbrole.go

```diff
@@ -16,55 +16,98 @@
 // specific language governing permissions and limitations
 // under the License.
 
-//	 DAL uses 4 database roles/users to perform all operations,
+//	DAL uses 4 database roles/users to perform all operations,
```

Comment:
Is it really correct to say that there will be 4 DB roles/users, considering there will actually be 8 of them?

Reply:
Let me update documentation too